config.m4

```diff
@@ -336,7 +336,7 @@ if test "$PHP_MONGODB" != "no"; then
     AC_SUBST(BSON_OS, 1)
 
     AC_SUBST(MONGOC_NO_AUTOMATIC_GLOBALS, 1)
-    AC_SUBST(MONGOC_ENABLE_MONGODB_AWS_AUTH, 0)
+    AC_SUBST(MONGOC_ENABLE_MONGODB_AWS_AUTH, 1)
```

Comment:
Does this PR depend on https://github.com/mongodb/mongo-php-driver/pull/1108? Presumably we cannot enable `MONGOC_ENABLE_MONGODB_AWS_AUTH` if KMS itself cannot be compiled (from either libmongoc or libmongocrypt).

I believe the most direct dependency would be on whether TLS is available.

Reply:
I'll run some tests without SSL. I'm under the impression that libmongoc will throw an error if enabling MONGODB-AWS without an SSL library. In that case, we may want to disable support for MONGODB-AWS as it's not an essential feature for most people.